### ISSUE TITLE
Big difference when exporting a model with Keras

### ISSUE THREAD

**Issue by sonfiree** [tensorflow/serving#1266]
Problems with accuracy when running on the server. Keras local results are much better

```
from tensorflow.python.saved_model import builder as saved_model_builder
from tensorflow.python.saved_model import tag_constants, signature_constants
from tensorflow.python.saved_model.signature_def_utils_impl import build_signature_def, predict_signature_def
from keras import backend as K
import tensorflow as tf
from keras.models import model_from_json

export_path = 'model/1'
sess = tf.Session()

K.set_session(sess)
K.set_learning_phase(1)

json_file = open(r'ex_model.json', 'r')
loaded_model_json = json_file.read()
json_file.close()
model.load_weights(r'model.10-0.69.h5' )

model.summary()


builder = saved_model_builder.SavedModelBuilder(export_path)

signature = predict_signature_def(inputs={'input_image': model.get_layer(name='the_input').input},
                                  outputs={'out':model.get_layer(name='out').output})

with K.get_session() as sess:
    builder.add_meta_graph_and_variables(sess=sess,
                                         tags=[tag_constants.SERVING],
                                         signature_def_map={'predict': signature})
    builder.save()

```

**Comment by sonfiree** [comment]
Weights need to freeze!
`K.set_learning_phase(0)`
